README.md

```diff
@@ -133,6 +133,10 @@ See [this issue](https://github.com/ansible-collections/community.general/issues
 
 ## Licensing
 
-GNU General Public License v3.0 or later.
+This collection is primarily licensed and distributed as a whole under the GNU General Public License v3.0 or later.
 
-See [COPYING](https://www.gnu.org/licenses/gpl-3.0.txt) to see the full text.
+See [LICENSES/GPL-3.0-or-later.txt](https://github.com/ansible-collections/community.general/blob/main/COPYING) for the full text.
+
+Parts of the collection are licensed under the [BSD 2-Clause license](https://github.com/ansible-collections/community.general/blob/main/LICENSES/BSD-2-Clause.txt), the [MIT license](https://github.com/ansible-collections/community.general/blob/main/LICENSES/MIT.txt), and the [PSF 2.0 license](https://github.com/ansible-collections/community.general/blob/main/LICENSES/PSF-2.0.txt).
+
+All files except changelog fragments (that will not be part of a release) have a machine readable `SDPX-License-Identifier:` comment denoting its respective license(s), or an equivalent entry in an accompanying ``.license`` file. This conforms to the `REUSE specification <https://reuse.software/spec/>`_.
```

Comment:
You may want to tweak this a bit now that there's the `dep5` file for the changelog fragments, but this is approved.

Reply:
How about:
```suggestion
All files have a machine readable `SDPX-License-Identifier:` comment denoting its respective license(s), or an equivalent entry in an accompanying `.license` file. Only changelog fragments (which will not be part of a release) are covered by a blanket statement in `.reuse/dep5` This conforms to the [REUSE specification](https://reuse.software/spec/).
```
?

(I also noticed I once again started writing RST in a MarkDown file... meh)

Reply:
> All files have a machine readable `SDPX-License-Identifier:` comment denoting its respective license(s), or an

You don't need a comma before `or`. I didn't notice that before. Besides that, LGTM :).